advanced_source/neural_style_tutorial.py

```diff
@@ -144,39 +136,34 @@ def imshow(tensor, title=None):
 imshow(content_img, title='Content Image')
 
 ######################################################################
-# Loss Functions
+# 손실 함수
 # --------------
-# Content Loss
+# 콘텐츠 손실(Content Loss)
 # ~~~~~~~~~~~~
 #
-# The content loss is a function that represents a weighted version of the
-# content distance for an individual layer. The function takes the feature
-# maps :math:`F_{XL}` of a layer :math:`L` in a network processing input :math:`X` and returns the
-# weighted content distance :math:`w_{CL}.D_C^L(X,C)` between the image :math:`X` and the
-# content image :math:`C`. The feature maps of the content image(:math:`F_{CL}`) must be
-# known by the function in order to calculate the content distance. We
-# implement this function as a torch module with a constructor that takes
-# :math:`F_{CL}` as an input. The distance :math:`\|F_{XL} - F_{CL}\|^2` is the mean square error
-# between the two sets of feature maps, and can be computed using ``nn.MSELoss``.
+# Content 손실은 각 계층에 대한 Content 거리의 가중치 버전을 나타냅니다.
+# 이 함수는 입력 :math:`X` 를 처리하는 레이어 :math:`L` 의 특징 맵 :math:`F_{XL}` 을 가져와서
+# 이미지 :math:`X` 와 Content 이미지 :math:`C` 사이의
+# 가중 콘텐츠 거리(weighted content distance) :math:`w_{CL}.D_C^L(X,C)` 를 반환합니다.
+# Content 거리를 계산하기 위해 Content 이미지(:math:`F_{CL}`)의 특징 맵을 함수에서 알고 있어야합니다.
+# :math:`F_{CL}` 을 입력으로 사용하는 생성자를 사용해 이 함수를 torch 모듈로 구현합니다.
+# 거리 :math:`\|F_{XL} - F_{CL}\|^2` 는 두개의 특징 맵 집합의 평균 제곱 오차이며 ``nn.MSELoss`` 를 사용하여 계산할 수 있습니다.
 #
-# We will add this content loss module directly after the convolution
-# layer(s) that are being used to compute the content distance. This way
-# each time the network is fed an input image the content losses will be
-# computed at the desired layers and because of auto grad, all the
-# gradients will be computed. Now, in order to make the content loss layer
-# transparent we must define a ``forward`` method that computes the content
-# loss and then returns the layer’s input. The computed loss is saved as a
-# parameter of the module.
+# Content 거리를 계산하기 위해 사용되는 합성곱 계층 바로 뒤에 Content 손실 모듈을 추가합니다.
+# 이렇게하면 입력이미지가 입력될 때마다 Content 손실이 원하는 레이어에서
+# 계산되고 autograd 때문에 모든 기울기가 계산됩니다.
+# 이제 Content 손실 계층을 만들기 위해 Content 손실을 계산한 다음 계층의 입력을 반환하는
+# ``forward`` 메소드를 정의해야합니다.
+# 계산된 손실은 모듈의 매개 변수로써 저장됩니다.
 #
 
 class ContentLoss(nn.Module):
 
     def __init__(self, target,):
         super(ContentLoss, self).__init__()
-        # we 'detach' the target content from the tree used
-        # to dynamically compute the gradient: this is a stated value,
-        # not a variable. Otherwise the forward method of the criterion
-        # will throw an error.
+        # 기울기를 동적으로 계산하는데 사용되는 tree로부터 타겟 Content를 `분리(detach)` 합니다.
```

Comment:
```suggestion
        # 기울기를 동적으로 계산하는데 사용되는 tree로부터 타깃 Content를 `분리(detach)` 합니다.
```
target 는 타깃으로 번역되는 듯 합니다